styles/mixins.scss

```diff
@@ -518,18 +518,14 @@ unpredictable css rules order in build */
 
 @mixin tab-panel() {
     display: flex;
-    flex-wrap: nowrap;
+    flex-wrap: wrap;
     justify-content: flex-start;
-    overflow: auto;
 
     &_centered {
-        display: flex;
         justify-content: center;
-        flex-wrap: wrap;
     }
 
     @media (max-width: map-get($gridBreakpoints, 'md')) {
-        display: flex;
         flex-wrap: nowrap;
         justify-content: flex-start;
```

Comment:
Is it supposed to be here without display flex?

Reply:
The `display: flex` property will be inherited from the base selector. So I think yes, it can be here without additional property setup.

Reply:
justify-content: flex-start; already exists in the base selector then)

Reply:
But it can be overriden by the &_centered's property assignment.